acsoo/addons.py

```diff
@@ -74,3 +75,25 @@ def addons_list_depends(ctx, exclude):
 
 
 addons.add_command(addons_list_depends, 'list-depends')
+
+
+@click.command()
+@click.option('--database')
+@click.option('--odoo-bin', 'odoo_bin', default='odoo',
+              type=click.Path(dir_okay=False, exists=True))
+@click.option('--odoo-config', 'odoo_config',
+              type=click.Path(dir_okay=False, exists=True))
+@click.option('--push', is_flag=True, default=False)
+@click.option('--branches', default='')
```

Comment:
I don't understand `--branches` (I would have expected an optional `--git-push-branch`).

Reply:
To avoid to generate translation for each branch ...

I will add --git-push-branch (It will be the CI_COMMIT_REF_NAME from gitlab or somthing like that)

Reply:
I understand. Then I would not have this `--branches`, this test can be done in `.gitlab-ci` if needed.